### ISSUE TITLE
Not an issue but a note on FFmpeg and GPU

### ISSUE THREAD

**Issue by ndrean** [dwyl/image-classifier#49]
I posted a [question on the Elixlirforum](https://elixirforum.com/t/use-ffmpeg-gpu-acceleration-with-bumblebee/61231): how to compile FFmpeg on Linux or OSX to use GPU? I target both local dev mode (if on Mac) and prod mode for Fly (Debian based image shipping FFmpeg). Let's see.

**Comment by ndrean** [comment]
I thank Jonathan for his clear response:

> It would be nice to have a package with NIF bindings to ffmpeg and precompiled binaries, so that we don’t rely on the global installation, if that’s what you mean. I don’t think that exists currently, and it’s not a priority : )

**Comment by LuchoTurtle** [comment]
Yeah, figured as such. There are loads of dependencies that follow the same pattern of relying on global installation of `ffmpeg` out there. Which makes sense, having to manually upload `ffmpeg` binaries shouldn't concern the package owner.

**Comment by ndrean** [comment]
FFMpeg has a huge API in terms of flags, for audoi, video, codecs etc.
Transforming this into a (NIFed) usable Elixir function seems like a huge task.
